conda_forge_tick/auto_tick.xsh

```diff
@@ -66,8 +66,20 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
                                    fork=fork,
                                    gh=gh)
 
-    # migrate the `meta.yaml`
     recipe_dir = os.path.join(feedstock_dir, 'recipe')
+    # if postscript/activate no noarch
+    suffixs = ['pre-unlink', 'post-link', 'pre-link']
+    exts = ['.bat', '.sh']
+    no_noarch_files = [
+        '{}-{}.{}'.format(attrs['feedstock_name'], suffix, ext)
```

Comment:
I don't think they need the feedstock name. See https://github.com/conda-forge/widgetsnbextension-feedstock/tree/master/recipe.